timescaledb-restore/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This dockerfile contains scripts and configurations which can be used to restore a postgres database using pgbackrest.
+# The image should only be use locally to restore CCD scan from backups.
+ARG timescale_image=timescale/timescaledb-ha:pg14.3-ts2.7.0-p0
+FROM ${timescale_image}
+
+USER root
+
+RUN apt-get update
+
```

Comment:
```suggestion
```

Remnant from earlier revision?

Reply:
No I need to change users to ably the chmod